docs/Project.toml

```
@@ -2,6 +2,7 @@
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
 HybridArrays = "1baab800-613f-4b0a-84e4-9cd3431bfbb9"
+JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
```

Is JSON still needed?
```suggestion
JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
```